mbed_lib.json

```diff
@@ -9,9 +9,13 @@
             "help": "Options are FAT and LITTLE",
             "required": false
         },
+        "filesystem-instances": {
+            "help": "Number of available filesystems (Default: 1)",
+            "value": 1
```

Comment:
Value should be same than PAL_NUMBER_OF_PARTITIONS. Please add some reference here or under documentation.